script.js

```diff
@@ -2,49 +2,73 @@
 //Declare a variable called 'name' that has the value of your name.
 
 /*code here*/
-
+var name="Jose";
 //#2
 //create an if statement that checks to see if your name is equal to 'Ben'
 //If yes, log 'yes'
 //otherwise log 'no'
 //note: use console.log to log the string
 
 /*code here*/
-
+if( name==='Ben')
+  Console.log("yes");
+else 
+  Console.log("no");
 //#3
 //create an array called 'favoriteFoods'.
 //fill it up with the names of several of your favorite foods
 
 /*code here*/
-
+var favoriteFoods=[
+  "Bananas","Apples","oranges","tika masala","pears", "California Burritos", "loquats"
```

Comment:
Great taste in food!

I'd recommend putting a space after commas and then putting the strings onto separate lines.
```
'bananas',
'apples',
'oranges',
'tika masala',
'etc',
```